tests/api-shortcuts.t

```diff
@@ -10,6 +10,9 @@ use strict;
 my $token = getTokenCookie();
 my $otherToken = getTokenCookie('user2');
 
+# Delete old shortcuts
+esPost("/tests_lookups/_delete_by_query?conflicts=proceed&refresh", '{ "query": { "match_all": {} } }');
```

Comment:
sure!

Reply:
Added tests for deleting shortcuts and actually delete the shortcuts. Also deleted the added _moloch_shared user so that the api-multies.t tests don't fail.